Library/Formula/repo.rb

```diff
@@ -1,9 +1,9 @@
 class Repo < Formula
   desc "Repository tool for Android development"
   homepage "https://source.android.com/source/developing.html"
-  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.21/repo"
-  version "1.21"
-  sha256 "44d1ac8b948e6154a495469dad05870dc6182712c0610da8d255b5471dd5bcb9"
+  url "https://raw.githubusercontent.com/android/tools_repo/v1.12.32/repo"
+  version "1.22"
+  sha256 "9907c36d63cf5222d73ab270fee5249dc84f6b4580cda03ddd7ac309f11b289e"
   revision 1
```

Comment:
Please remove the line with `revision 1`. Revisions are reset when we upgrade the version; so this line is not necessary anymore.